src/sections/pricing_table.liquid

```diff
@@ -1,24 +1,30 @@
 <section class="pricing-table {% if section.settings.background_color_preset != 'none' and section.settings.background_color_preset != blank %}{{ section.settings.background_color_preset | prepend: 'section-color-preset__' }}{% endif %}" data-preview-item="pricing-table">
   <div class="container">
-    <article class="section__content">
-      {% if section.settings.heading != blank %}
-        <h3 class="section__heading">{{ section.settings.heading }}</h3>
-        {% if section.settings.subheading != blank %}
-          <h4 class="section__subheading">{{ section.settings.subheading }}</h4>
+    <div class="row">
+      <article class="section__headings">
+        {% if section.settings.heading != blank %}
+          <h3 class="section__heading">{{ section.settings.heading }}</h3>
+          {% if section.settings.subheading != blank %}
+            <h4 class="section__subheading">{{ section.settings.subheading }}</h4>
+          {% endif %}
         {% endif %}
-      {% endif %}
-      {% if section.blocks != empty %}
-        <ul class="pricing-table__list">
-          {% for block in section.blocks %}
-            <li class="pricing-table__list-item">
-              <h4>{{ block.settings.po_price }}</h4>
-              <h4>{{ block.settings.po_type }}</h4>
-              <a href="{{ block.settings.po_button.links.add_to_cart }}" class="button button-primary">Buy Now</a>
-            </li>
-          {% endfor %}
-        </ul>
-      {% endif %}
-    </article>
+      </article>
+      <article class="section__content">
+        {% if section.blocks != empty %}
+          <ul class="pricing-table__list">
+            {% for block in section.blocks %}
+              <li class="pricing-table__list-item">
+                <div class="pricing-table__list-item-details">
+                  <h4>{{ block.settings.po_price }}</h4>
```

Comment:
lolol, it's picky but can you change this to `block.settings.pricing_option_price` pls? Same with below